CMakeLists.txt

```diff
@@ -15,10 +15,20 @@ find_package(builtin_interfaces REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
 add_library(iio SHARED IMPORTED) # or STATIC instead of SHARED
+
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
+    set_target_properties(iio PROPERTIES
```

Comment:
This is guessing where libiio is.
Cmake allows more enhanced ways to do this. One is using: find_library()